src/Common/src/System/Globalization/FormatProvider.Number.cs

```diff
@@ -1364,14 +1359,14 @@ internal static unsafe string NumberToStringFormat(NumberBuffer number, string f
 
                         int[] groupDigits = info.NumberGroupSizes;
 
-                        int groupSizeIndex = 0;     // index into the groupDigits array.
+                        int groupSizeIndex = 0;     // Index into the groupDigits array.
                         int groupTotalSizeCount = 0;
-                        int groupSizeLen = groupDigits.Length;    // the length of groupDigits array.
+                        int groupSizeLen = groupDigits.Length;    // The length of groupDigits array.
                         if (groupSizeLen != 0)
-                            groupTotalSizeCount = groupDigits[groupSizeIndex];   // the current running total of group size.
+                            groupTotalSizeCount = groupDigits[groupSizeIndex];   // The current running total of group size.
                         int groupSize = groupTotalSizeCount;
 
-                        int totalDigits = digPos + ((adjust < 0) ? adjust : 0); // actual number of digits in o/p
+                        int totalDigits = digPos + ((adjust < 0) ? adjust : 0); // Actual number of digits in o/p
```

Comment:
FYI, for future cleanup, there's no need to change comments like this to start with a capital... it's fine having short descriptions that aren't full sentences.